### DISCUSSION TITLE
How to use Ant Design

### DISCUSSION THREAD

**Discussion by ArtFlag** [facebook/docusaurus#4114] [Q&A]
I'm trying to use Ant design in my SPA but importing their css in an mdx file overrides a few styles used in the classic theme. 
I do not have this problem if I import only a given antd component's css insead of the whole css. 

What is the correct way to use antd or other similar packages? Should I import the CSS via a plugin somehow?

**Comment by slorber** [comment]
If the Antd classes conflict with Infima classes, apart not importing the conflicting classes I don't have any other solution.

This is not really a problem that we can solve at Docusaurus level. 
All frameworks using global utility classes can conflict with each others
